cmake/compiler_config.cmake

```diff
@@ -106,7 +106,7 @@ function(mayaUsd_compile_config TARGET)
         if(IS_LINUX)
             target_compile_definitions(${TARGET}
                 PRIVATE
-                    _GLIBCXX_USE_CXX11_ABI=0 # USD is built with old ABI
+                    _GLIBCXX_USE_CXX11_ABI=$<IF:$<BOOL:${MAYA_LINUX_BUILT_WITH_CXX11_ABI}>,1,0>
```

Comment:
Set the cxx11 abi to same value as the Maya we are compiling with. The two must match.